### ISSUE TITLE
automatic peak-picking / SINO measurement

### ISSUE THREAD

**Issue by djeanner** [cheminfo/nmr-displayer#39]
If you want to introduce an automatic peak picking, you may want to consider the automatic measurement of the SINO. Than you take peaks for extrema that are above&below 5xnoise level.  I wrote a paper on a nice algorith to determine  SINO automatically in 1D and 2D spectra https://onlinelibrary.wiley.com/doi/full/10.1002/mrc.4882

**Comment by lpatiny** [comment]
Yes excellent idea. Could you contribute with those algorithms in the project:
https://github.com/mljs/spectra-processing
And in particular in the folder: https://github.com/mljs/spectra-processing/tree/master/src/xy
You can calculate information from an object containing x and y array.
We can meet so that you develop it with us and we show you best practices in javascript so that you can contribute easily in the future.

**Comment by lpatiny** [comment]
@djeanner Anything new about the ? 

You may also directly add the code as a filter in the following folder and make a pull request:

https://github.com/cheminfo/nmr-displayer/tree/master/src/data/data1d/filter1d

**Comment by djeanner** [comment]
Tu m’avais dit que tu m’expliquerais...
Je parlais de novembre pour cela - nous y sommes. Est-ce qu’il y a un moment de la semaine / journée qui te convient pour que je passe à Ls?
Amicalement,
aminé

—————————————
e-Research/DLCM
University of Geneva
damien.jeannerat@unige.ch<mailto:damien.jeannerat@unige.ch>
—————————————
NMReDATA Initiative
nmredata.org
—————————————
NMRprocess.ch
damien.jeannerat@protonmail.com

On 29 Oct 2019, at 10:24, Luc Patiny <notifications@github.com<mailto:notifications@github.com>> wrote:


@djeanner<https://github.com/djeanner> Anything new about the ?

You may also directly add the code as a filter in the following folder and make a pull request:

https://github.com/cheminfo/nmr-displayer/tree/master/src/data/data1d/filter1d
